### PR TITLE
Created like model

### PR DESCRIPTION
**Pull request by Khaledelabady11**

### CODE REVIEW

**Comment by Khaledelabady11** [comment]
ref : https://stackoverflow.com/questions/61430073/please-explain-the-has-many-through-source-rails-association

**Comment by m7madmagdy** [comment]
Nice 👍❤️

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class LikesController < HtmlController
+
+  def create
+    @like = Like.new(article_id: params[:article_id], user_id: current_user.id)
+    if @like.save
+      redirect_to @like.article
+    end
+  end
+end
```
Comment (a7madM) [inline app/controllers/likes_controller.rb:9]: this should be in APIs not HTML !

Comment (Khaledelabady11) [reply app/controllers/likes_controller.rb]: done

File: app/controllers/api/likes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class LikesController < ApiController
```
Comment (a7madM) [inline app/controllers/api/likes_controller.rb:1]: add api module in top

Comment (m7madmagdy) [reply app/controllers/api/likes_controller.rb]: Done

**Pull request merged**